samples/core/binaries/main.c

```diff
@@ -159,7 +159,7 @@ int main(int argc, char *argv[])
     cl_platform_id platform;
     cl_device_id device;
     cl_context context;
-    cl_command_queue queue;
+    cl_command_queue queue = NULL;
```

Comment:
It seems odd to initialize the command queue but not the platform, device, or context.

Reply:
This change was also prompted by a warning fix on Android. Now I looked at it again and realized, that the warning pointed out a valid bug in the resource release logic. That I fixed and this change was reverted.